### ISSUE TITLE
QhullError when trying to measure size or shape of single-slice label object on 3D image

### ISSUE THREAD

**Issue by loicsauteur** [kevinyamauchi/morphometrics#21]
Hi @kevinyamauchi and @haesleinhuepf 

Cool plugin!

(Firstly, I had some trouble installing it on my Mac, but after installing `pyopencl` via Anaconda-Gui and `hdbscan` using the `conda install -c conda-forge hdbscan` command, it worked.)

I wanted to try it to measure properties in 3D nuclei images with corresponding 3D segmentation/label image. 
My data is very anisotropic, hence some nuclei may have only one slice-label, while others may span several slices.

As soon as I have one label item drawn only one slice I get this error, when trying to measure regionprops *size* or *shape*:
```python
QhullError: QH6013 qhull input error: input is less than 3-dimensional since all points have the same x coordinate    0

While executing:  | qhull i Qt
Options selected for Qhull 2019.1.r 2019/06/21:
  run-id 848841556  incidence  Qtriangulate  _pre-merge  _zero-centrum
  _max-width 12  Error-roundoff 1.7e-14  _one-merge 1.2e-13
  _near-inside 5.8e-13  Visible-distance 3.3e-14  U-max-coplanar 3.3e-14
  Width-outside 6.7e-14  _wide-facet 2e-13  _maxoutside 1.3e-13
```

Of course it makes sense that an object drawn on a single slice may not have a 3D hull. When I remove these single-slice objects everything works as expected.

Just wanted to report this user-case. Maybe it is not relevant for this plugin, because one could do the more basic stuff using just `skimage.measure.regionprops` function. Or maybe you have an idea to deal with such exceptions?
Hope my description is clear enough without example code.

napari-info:
```
napari: 0.4.13
Platform: macOS-10.16-x86_64-i386-64bit
System: MacOS 11.6.5
Python: 3.8.12 | packaged by conda-forge | (default, Oct 12 2021, 21:50:38) [Clang 11.1.0 ]
Qt: 5.15.2
PyQt5: 5.15.5
NumPy: 1.21.5
SciPy: 1.8.0
Dask: 2021.10.0
VisPy: 0.9.4

OpenGL:
- GL version: 2.1 Metal - 71.7.1
- MAX_TEXTURE_SIZE: 16384

Screens:
- screen 1: resolution 2560x1440, scale 1.0
- screen 2: resolution 1440x900, scale 2.0

Plugins:
- bioformats: 0.2.1
- console: 0.0.4
- napari-clusters-plotter: 0.3.0
- napari-features: 0.1.4
- napari-nikon-nd2: 0.1.3
- napari-tools-menu: 0.1.11
- napari_skimage_regionprops1: 0.4.1
- napari_skimage_regionprops2: 0.4.1
- ome-types: 0.2.9
- omero: 0.1.1
- scikit-image: 0.4.13
- stl: 0.0.6
- svg: 0.1.6
```

**Comment by kevinyamauchi** [comment]
Hey @loicsauteur ! It's nice to see you here. Thanks for the heads up about the installation issues. I'll have a look on my mac and see if that can be simplified.

Let us know about the scikit-image version as Robert suggested - hopefully an update will fix it!

**Comment by kevinyamauchi** [comment]
For reference, the skimage version that includes the fix linked by @haesleinhuepf is 0.19.0

https://github.com/scikit-image/scikit-image/releases/tag/v0.19.0

**Comment by kevinyamauchi** [comment]
Great! Thanks for the feedback, @loicsauteur ! I will close this issue for now. Please feel free to reopen if if the issue pops back up.
